Cython/Includes/libcpp/vector.pxd

```diff
@@ -73,9 +98,11 @@ cdef extern from "<vector>" namespace "std" nogil:
         void pop_back()
         void push_back(T&) except +
         reverse_iterator rbegin()
-        const_reverse_iterator const_rbegin "crbegin"()
```

Comment:
I believe this was incorrect, since the `const_*` versions appear to be a hack since cython does not understand when to use const vs non const versions this function. Even though now that the iterators work as expected I would personally prefer:
```cython
cdef test(const vector[int]& x):
     cdef vector[int].const_iterator iter = x.begin()
```
or if C++11 is used
```cython
cdef test(const vector[int]& x):
     iter = x.cbegin()
```

over
```cython
cdef test(const vector[int]& x):
     cdef vector[int].const_iterator iter = x.const_begin()
```
or
```cython
cdef test(const vector[int]& x):
     iter = x.const_begin()
```

since it is closer to the actual C++ code you would write